src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const minDiscont = 3;
```

Comment:
Consider renaming `minDiscont` to `minDiscount` for better readability and to avoid confusion.